samples/http-client-examples/client.mts

```diff
@@ -0,0 +1,90 @@
+import {
```

Comment:
[req] I think we must ensure that the samples are not degrading over time. Could you perhaps add a test which starts the server and executes the clients. No big assertions just that the server and client start.